search/includes/classes/class-search.php

```diff
@@ -1109,13 +1099,13 @@ public function is_url_query_cacheable( string $url, $args ): bool {
 	 * @param mixed $request
 	 * @param mixed $response
 	 * @param array $query
-	 * @param string $statsd_prefix
+	 * @param string $unused unused
 	 * @param string $type
 	 * @param mixed $cached_request
 	 * @param string $query_method
 	 * @return void
 	 */
-	public function ep_handle_failed_request( $request, $response, $query, $statsd_prefix, $type, $cached_request, $query_method ) {
+	public function ep_handle_failed_request( $request, $response, $query, $unused, $type, $cached_request, $query_method ) {
```

Comment:
Should we just change the function signature since we won't be needing $statsd_prefix anymore?

Reply:
I doubt we would but updating method on a public method is a bit risky.

Reply:
It's not being used anywhere. Also, can you detail what type of testing is needed to be done on this removal?

Reply:
Addressed